--- distinfo.or 2023-11-29 23:10:24.852421000 -0500
+++ distinfo    2023-11-29 23:10:54.553821000 -0500
@@ -143,4 +143,6 @@
 SIZE (tg123-websockify-nginx-module-c11bc9a_GH0.tar.gz) = 14646
 SHA256 (openresty-xss-nginx-module-v0.06_GH0.tar.gz) = 0b12bbc53a41f3e3d6df419c173b8c87434be3e6cd255a8193aa91345a2de6cf
 SIZE (openresty-xss-nginx-module-v0.06_GH0.tar.gz) = 12448
+SHA256 (kjdev-nginx-auth-jwt-40c9191_GH0.tar.gz) = 0705841c5b281699788f2eecda72962f4082dedf618939b128ef5f997e50e854
+SIZE (kjdev-nginx-auth-jwt-40c9191_GH0.tar.gz) = 61300